debian/control

```diff
@@ -14,23 +14,35 @@ Build-Depends: debhelper (>= 10),
           libncursesw5-dev,
           libdaemon-dev,
           libjansson-dev,
-          python3-pip
+          python3-pip,
+          dh-python,
+          python3-setuptools,
+          python3
 Standards-Version: 4.1.2
 Section: libs
 Homepage: https://developer.ridgerun.com/wiki/index.php?title=GStreamer_Daemon_-_GStreamer_Daemon_Basics
 
 Package: gstd
 Architecture: any
-Multi-arch: same
+Multi-arch: allowed
 Depends: ${shlibs:Depends},
-         ${misc:Depends}
+         ${misc:Depends},
+         adduser
 Description: GStreamer Daemon 
-  GStreamer Daemon, also called gstd, is a GStreamer 
-  framework for controlling audio and video streaming 
-  using TCP connection messages. Qt applications, web 
-  interfaces, and GStreamer element test suites show 
-  different ways the framework can be utilized. 
-  GStreamer Daemon is gst-launch on steroids where you 
-  can create a GStreamer pipeline, play, pause, change 
-  speed, skip around, and even change element parameter 
-  settings all while the pipeline is active.
+ GStreamer Daemon, also called gstd, is a GStreamer
+ framework for controlling audio and video streaming
+ using TCP connection messages. Qt applications, web
+ interfaces, and GStreamer element test suites show
+ different ways the framework can be utilized.
+ GStreamer Daemon is gst-launch on steroids where you
+ can create a GStreamer pipeline, play, pause, change
+ speed, skip around, and even change element parameter
+ settings all while the pipeline is active.
+
+Package: pygstc
+Architecture: all
+Depends: ${misc:Depends},
+         ${python3:Depends},
+         python3
```

Comment:
Is there a way to specify the version of python? A.f.a.i.k, we require 3.5 as the minimum

Reply:
Yes, I can add `python3 (>=3.5)` but I am not sure what is the min version supported. Do you know the exact version or is `>=3.5` fine?

Reply:
I think a greater or equal than 3.5 should be good enough.